src/routers/createConfigGetter.js

```diff
@@ -17,8 +17,8 @@ import type {
 } from '../TypeDefinition';
 
 function applyConfig(
-  configurer: ?NavigationScreenConfig,
-  prevOptions: *,
+  configurer: ?NavigationScreenConfig<Object>,
+  prevOptions: Object,
```

Comment:
can we use a more detailed type than Object? Even if we just import the same thing from typeDefinition